lib/internal/Magento/Framework/View/Test/Unit/Asset/Bundle/ManagerTest.php

```diff
@@ -118,6 +118,13 @@ public function testAddAssetWithExcludedFile()
         $this->asset->expects($this->atLeastOnce())
             ->method('getFilePath')
             ->willReturn('source/file.min.js');
+        $this->asset->expects($this->once())
+            ->method('getPath')
+            ->willReturn('');
+        $dirRead->expects($this->once())
+            ->method('getAbsolutePath')
+            ->with('')
+            ->willReturn(true);
```

Comment:
I believe the `getAbsolutePath` method should return a path, not a boolean, right?

Reply:
Ok, I made changes to some string. Thanks for your help.